zubhub_frontend/zubhub/src/views/create_team/step3/Step3.jsx

```diff
@@ -71,47 +70,45 @@ export default function Step3({ updateSelectedProjects, formik, handleBlur, ...p
             classes={classes}
             common_classes={commonClasses}
             inputOrder={4}
-            fieldLabel={('createTeam.3rdpage.title')}
+            fieldLabel={'createTeam.3rdpage.title'}
           />
         </Grid>
         <Grid item xs={12} className={commonClasses.commonClasses}>
           <Paper className={classes.profileLowerStyle}>
             <Grid container spacing={3}>
               {Array.isArray(projects) &&
-                projects.map((project) => (
-                  <Grid
-                    item
-                    xs={12}
-                    sm={6}
-                    md={6}
-                    className={classes.projectGridStyle}
-                    align="center"
-                  >
+                projects.map(project => (
+                  <Grid item xs={12} sm={6} md={6} className={classes.projectGridStyle} align="center">
                     <div style={{ position: 'relative', marginLeft: '16px', marginRight: '16px' }}>
-                    <Project
-                      project={project}
-                      t={t}
-                      handleProjectClick={handleProjectClick}
-                    />
-                    <div className={clsx(classes.overLay, selectedProjects.some(p => p.id === project.id) && classes.blurOverLay)}
-                      onClick={()=> selectProjects(project)}  style={{
-                        display: "flex",
-                        flexDirection: "column",
-                        justifyContent: "center",
-                        alignItems: "center",
-                      }}>
-                          {selectedProjects.some(p => p.id === project.id) ? 
+                      <Project project={project} t={t} handleProjectClick={handleProjectClick} />
+                      <div
+                        className={clsx(
+                          classes.overLay,
+                          selectedProjects.some(p => p.id === project.id) && classes.blurOverLay,
+                        )}
+                        onClick={() => selectProjects(project)}
+                        style={{
+                          display: 'flex',
+                          flexDirection: 'column',
+                          justifyContent: 'center',
+                          alignItems: 'center',
+                        }}
+                      >
+                        {selectedProjects.some(p => p.id === project.id) ? (
                           <span
                             style={{
```

Comment:
remove inline style and move to the *Style.js file. This wasn't introduced by you but please help and remove it